notebook/data_exploration/dataset_stats.ipynb

```diff
@@ -0,0 +1,262 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dataset Statistics\n",
+    "This notebook keeps track of various statistics about the compendium dataset like how many samples are labled, which diseases are present, etc.\n",
+    "\n",
+    "To skip straight to the stats, click [here](#stats)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from collections import Counter\n",
+    "import json\n",
+    "import os\n",
+    "import sys\n",
+    "\n",
+    "# __file__ isn't defined for jupyter notebooks\n",
+    "curr_path = os.path.dirname(os.path.abspath(os.path.abspath('')))\n",
+    "whistl_path = os.path.join(curr_path, os.pardir, 'whistl')\n",
+    "sys.path.append(whistl_path)\n",
+    "import utils"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "map_file = os.path.join(curr_path, os.pardir, 'data', 'sample_classifications.pkl')\n",
```

Comment:
Maybe helpful to add a comment in the function for what the format of the data is for this map_file is